tests/Functional/Schema/CustomIntrospectionTest.php

```diff
@@ -29,7 +29,7 @@ public static function setUpBeforeClass(): void
             self::markTestSkipped('Skip on Oracle');
         }
 
-        Type::addType(MoneyType::NAME, MoneyType::class);
+        Type::getTypeRegistry()->register(MoneyType::NAME, new MoneyType);
```

Comment:
The API could be as simple as this:

```suggestion
        Type::addType(MoneyType::NAME, new MoneyType());
```